homeassistant/components/concord232/alarm_control_panel.py

```diff
@@ -89,32 +93,38 @@ def __init__(self, url, name, code, mode):
         self._alarm = concord232_client.Client(self._url)
         self._alarm.partitions = self._alarm.list_partitions()
 
-    def update(self) -> None:
-        """Update values from API."""
+    @property
+    def alarm_status(self) -> str:
+        """Get current status from API."""
         try:
             part = self._alarm.list_partitions()[0]
         except requests.exceptions.ConnectionError as ex:
             _LOGGER.error(
                 "Unable to connect to %(host)s: %(reason)s",
                 {"host": self._url, "reason": ex},
             )
-            return
+            return STATE_UNAVAILABLE
```

Comment:
`STATE_UNAVAILABLE` is not a direct state that should be used. The `unavailable` state is a result of implementing the `available()` property.